charts/ecosystem/values.yaml

```diff
@@ -16,6 +16,12 @@ externalHostname: "example.com"
 galasaVersion: "0.36.0"
 #
 #
+#
+# Name of the service being used
```

Comment:
Would be good to expand on this description since it might confuse people installing the helm chart and because it isn't immediately obvious where this is used. Maybe something that mentions this being the name that gets displayed on the web UI?